lxd/apparmor/rsync.go

```diff
@@ -29,6 +29,9 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
 
   unix (connect, send, receive) type=stream,
 
+  # PowerFlex storage driver specific for NVMe/TCP mode
+  network inet tcp,
```

Comment:
Do we need `network inet6 tcp,` too for IPv6-based deployments?

Reply:
For coarse grain rules like blanket authorization for TCP, I think we should always do it for the 2 inet families. There is no security difference to a TCP connection over IPv6 or over IPv4 :)

Reply:
Agreed. But i don't think we should allow either in this case.

Reply:
Oh that's right, thanks for pointing that out. Neither `rsync` nor `qemu-img` should have anything related to powerflex in their Apparmor profile.

In the `rsync` case, I see that no network communication is/was permitted. Presumably that's because it all goes over the WebSocket transport accessed via a Unix socket?

Reply:
That's correct. It seems likely this is a kernel bug that @mihalicyn us going to look into

Reply:
That's correct. It seems likely this is a kernel bug that @mihalicyn us going to look

Reply:
And in this case there isnt even a web socket involved, its a local rsync to an nvme over tcp mounted drive